checker/src/main/java/org/checkerframework/checker/signedness/SignednessUtil.java

```diff
@@ -212,10 +212,12 @@ public static ByteBuffer putUnsignedLong(ByteBuffer b, int i, @Unsigned long ulo
     }
 
     /**
-     * Reads an unsigned byte from the RandomAccessFile f. This method is a wrapper around {@link
-     * java.io.RandomAccessFile#readByte() readByte()}, but assumes the output should be interpreted
-     * as unsigned.
+     * @deprecated This method is no longer used as it is defined in Java 8. Reads an unsigned byte
```

Comment:
This is using the wrong syntax for the `@deprecated` Javadoc tag.  It should appear after the main text rather than incorporating it.

The `@deprecated` tag's text should indicate the exact method that clients should call.
By convention, there should be a // comment after the `@Deprecated` annotation with that same information.